app/views/audit.scala.html

```diff
@@ -782,10 +792,34 @@ <h1 class="text-center bold">@Messages("surface.problem")</h1>
         <div class="row">
             <div class="col-md-12">
                 <p>@Html(Messages("audit.tutorial.end.1"))</p>
-                <p>
-                    <img src='@routes.Assets.at("javascripts/SVLabel/img/onboarding/ExamplePictures02.png")' id="onboarding-example-image-2"
-                    alt="Examples of accessibility attributes: curb ramps, missing curb ramps, obstacles in path, and surface problems.">
-                </p>
+                
+                <table class="tutorial-table">
+                    <tr>
+                        <th class="tutorial-table-cell">@Html(Messages("audit.tutorial.accessible"))</th>
+                        <th class="tutorial-table-cell">@Html(Messages("audit.tutorial.inaccessible"))</th>
+                    </tr>
+                    <tr>
+                        <td colspan=2><img src='@routes.Assets.at("javascripts/SVLabel/img/onboarding/ExamplePictures02_top.png")' id="onboarding-example-image-2"
```

Comment:
Here is another small example of splitting the HTML onto multiple lines appropriately. Looking for the following (same thing on a few other lines in this file)
```
<td colspan=2>
    <img src='@routes.Assets.at("javascripts/SVLabel/img/onboarding/ExamplePictures02_top.png")' id="onboarding-example-image-2" alt="@Messages("audit.tutorial.examples")">
</td>
```

Another very small point is that I'd probably always put the `id` attribute first.

But I do want to say that this HTML is looking SO much cleaner. It looks really really good, this is the only small thing I found.

Reply:
Thank you for working with me on this! All issues that will not be a problem in the future. Bad habits I developed from trying to code my own personal websites/projects quickly